benchmarks/src/main/java/org/elasticsearch/benchmark/routing/allocation/Allocators.java

```diff
@@ -72,17 +72,14 @@ private Allocators() {
     }
 
     public static AllocationService createAllocationService(Settings settings) {
-        return createAllocationService(settings, new ClusterSettings(Settings.EMPTY, ClusterSettings.BUILT_IN_CLUSTER_SETTINGS));
-    }
-
-    public static AllocationService createAllocationService(Settings settings, ClusterSettings clusterSettings) {
         return new AllocationService(
-            defaultAllocationDeciders(settings, clusterSettings),
+            defaultAllocationDeciders(settings, new ClusterSettings(Settings.EMPTY, ClusterSettings.BUILT_IN_CLUSTER_SETTINGS)),
```

Comment:
NIT: could be simplified to
```suggestion
            defaultAllocationDeciders(settings, createBuiltInClusterSettings()),
```